WifiDirectTest/.gitignore

```diff
@@ -0,0 +1,6 @@
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
```

Comment:
Can you please add .idea/ and **/*.iml to the .gitignore file so that we don't have extraneous intellij files checked in?